packages/nodes-base/nodes/KoboToolbox/KoboToolboxTrigger.node.ts

```diff
@@ -0,0 +1,158 @@
+	IWebhookResponseData,
+	LoggerProxy as Logger,
+} from 'n8n-workflow';
+
+import {
+	formatSubmission,
+	koboToolboxApiRequest,
+	parseStringList
+} from './GenericFunctions';
+
+import {
+	formattingOptions,
+} from './descriptions';
+
+export class KoboToolboxTrigger implements INodeType {
+	description: INodeTypeDescription = {
+		displayName: 'KoboToolbox Trigger',
+		name: 'koboToolboxTrigger',
+		icon: 'file:koboToolbox.svg',
+		group: ['trigger'],
+		version: 1,
+		description: 'Process KoboToolbox submissions',
+		defaults: {
+			name: 'KoboToolbox Trigger',
+			color: '#64C0FF',
+		},
+		inputs: [],
+		outputs: ['main'],
+		credentials: [
+			{
+				name: 'koboToolboxApi',
+				required: true,
+			},
+		],
+		webhooks: [
+			{
+				name: 'default',
+				httpMethod: 'POST',
+				responseMode: 'onReceived',
+				path: 'webhook',
+			},
+		],
+		properties: [
+			{
+				displayName: 'Form ID',
+				name: 'asset_uid',
+				type: 'string',
+				required: true,
+				default:'',
+				description:'Form id (e.g. aSAvYreNzVEkrWg5Gdcvg)',
+			},
+			{...formattingOptions},
+		],
+	};
+
+	// @ts-ignore
+	webhookMethods = {
+		default: {
+			async checkExists(this: IHookFunctions): Promise<boolean> {
+				const webhookData = this.getWorkflowStaticData('node');
+				const webhookUrl = this.getNodeWebhookUrl('default');
+				const asset_uid = this.getNodeParameter('asset_uid') as string; //tslint:disable-line:variable-name
+				const {results: webhooks} = await koboToolboxApiRequest.call(this, {
+					url: `/api/v2/assets/${asset_uid}/hooks/`,
+				});
+				for (const webhook of webhooks || []) {
+					if (webhook.endpoint === webhookUrl && webhook.active === true) {
+						webhookData.webhookId = webhook.uid;
+						return true;
+					}
+				}
+				return false;
+			},
+
+			async create(this: IHookFunctions): Promise<boolean> {
+				const webhookData = this.getWorkflowStaticData('node');
+				const webhookUrl = this.getNodeWebhookUrl('default');
+				const asset_uid = this.getNodeParameter('asset_uid') as string; //tslint:disable-line:variable-name
+
+				// First check if one exists but is inactive
+				const { results: webhooks } = await koboToolboxApiRequest.call(this, {
+					url: `/api/v2/assets/${asset_uid}/hooks`,
+				});
+				for (const webhook of webhooks) {
+					if (webhook.endpoint === webhookUrl) {
+						await koboToolboxApiRequest.call(this, {
+							method: 'PATCH',
+							url: `/api/v2/assets/${asset_uid}/hooks/${webhook.uid}`,
+							body: {
+								active: true,
+							},
+						});
+						webhookData.webhookId = webhook.uid;
+						return true;
+					}
+				}
+
+				// No existing hook found, create one
+				const response = await koboToolboxApiRequest.call(this, {
+					method: 'POST',
+					url: `/api/v2/assets/${asset_uid}/hooks/`,
+					body: {
+						name: `n8n Automatic Webhook`,
+						endpoint: webhookUrl,
+						email_notification: true,
+					},
+				});
+				Logger.debug('KoboToolboxTriggerCreate', response);
```

Comment:
The create method will only be called if checkExist returns false. You already validated this on checkExist function. You do not need it here.